quarkus/tests/integration/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -58,50 +72,75 @@ private GenericContainer getKeycloakContainer() {
         }
         fetchDockerfile();
         return new GenericContainer(
-                new ImageFromDockerfile()
+                new ImageFromDockerfile("keycloak.x-under-test", false)
```

Comment:
Not entirely sure if this shouldn't have a try-with-resources block. Or the caller (start()). In general, GenericContainer implements AutoCloseable. We should use it. Let's talk about this ;)

Reply:
we can't use a `try-with-resources` since the container is expected to stay live during the tests and we can't close the the resource in the `start` method